dev/bots/test/analyze-sample-code-test-input/known_broken_documentation.dart

```diff
@@ -41,3 +41,16 @@
 /// )
 /// ```
 /// {@end-tool}
+///
+/// {@tool sample}
+/// ```dart
+/// const Text('Poor wandering ones!')
+/// ```
+/// {@end-tool}
+///
+/// {@tool sample}
+/// ```dart
+/// const text0 = Text('Poor wandering ones!');
+/// const text1 = Text('Poor wandering ones!');
```

Comment:
```suggestion
/// const text1 = Text('Poor wandering ones!');
/// const text2 = const   _Text('Poor wandering ones!');
```

Reply:
I was under the impression that `const text2 = const   _Text('Poor wandering ones!');` violates the style guide. Surprisingly the analyzer didn't complain.